src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -49,7 +49,7 @@ public Integer doCall() throws IOException {
 						+ System.getenv("GITPOD_WORKSPACE_URL") + "#" + project.getAbsolutePath() + "\n\n");
 			} else {
 				List<String> optionList = new ArrayList<>();
-				optionList.add(liveeditor);
+				optionList.addAll(Arrays.asList(liveeditor.split(" ")));
```

Comment:
this wont work on windows where spaces in paths are common. i think we need to be smarter.

Reply:
Good point, didn't think about it. I can't think of a way to figure out which part is the path to the binary file and which part is arguments though in such a scenario, by "just looking" at the string. I think we should start "consuming" the string and checking if it matches a path and stop once we reach a file on the filesystem, then the rest should be treated as arguments.

This is again problematic though since you might have two files with same prefix, like:
* `this is my exe`
* `this is my exe 2`

So how would you know if the user wants to run `this is my exe` with `2` as a parameter or `this is my exe 2` with no parameters, when given `--live="this is my exe 2"`.